docs/users.md

```diff
@@ -522,6 +530,63 @@ vcluster-0                                          2/2     Running   0
 
 The nginx pod is the one with the name `nginx-x-default-x-vcluster`.
 
+## Working with an external control plane
+
+In this section, we will show an example of creating an external control plane to integrate
+a kind cluster named `ext1` into the same Docker network as your Kubeflex hosting cluster. This setup 
+requires that both clusters are accessible from each other within the shared Docker network.
```

Comment:
Isn't it true that Docker supports multiple networks, so "the" shared network is not a general statement? Isn't it true that the real constraint here is that the "external" cluster and the KubeFlex hosting cluster are both on the same Docker network, regardless of which one that is?